config/nginx.conf

```diff
@@ -9,19 +9,4 @@ server {
         uwsgi_pass_request_headers on;
         uwsgi_pass_request_body on;
     }
-
-    location @webpack {
-        proxy_pass http://watch:8078;
-    }
-
-    location ^~ /static/(.*$) {
-        try_files $uri $uri/ staticfiles/$1 staticfiles/$1/ @webpack =404;
-    }
-
-    location /static/debug_toolbar/ {
-        include uwsgi_params;
-        uwsgi_pass web:8077;
-        uwsgi_pass_request_headers on;
-        uwsgi_pass_request_body on;
-    }
```

Comment:
If we leave the `/static` location then it is necessary to have the debug_toolbar special case, otherwise all assets under that path will return a 404.